tests/python/create.py

```diff
@@ -55,8 +55,16 @@ def parse_args():
 
     await create_accounts(root_kp, kps, channel_builders, args.horizon, STARTING_BALANCE)
 
-    for kp in kps:
-        print(kp.secret_seed)
+    if args.json_output:
+        print("{" + '"keypairs"' + ":[{" , end = '')
+        for kp in kps:
+            print('{"address:"' + '"' + kp.public_address + '"' , end = '')
+            print('"seed:"' + '"' + kp.secret_seed + '"}' , end = '')
+        print("]}" , end = '')
```

Comment:
You can quickly build a dictionary and serialize it instead. Clearer code and less error prone:
```python
out = []
for kp in kps:
    out.append({"address": kp.public_address, "seed": kp.secret_seed})
print(json.dumps(out, indent=True))
```
json dumping with `indent=True` will print it in human readable format. It's not necessary for the load testing app but I think it's better that we'll be able to scan the files with our eyes as well.